04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md

```diff
@@ -9,30 +9,6 @@ tags:
 
 #placeholder/description 
 
-## MOC
-
```

Comment:
I'd also keep this one as a manual MOC to be refactored (probably needs headings to split the guides into useful things).

Reply:
Done: https://github.com/obsidian-community/obsidian-hub/commit/2b359404646307d3de317f7a9ae1f60007790b13